src/ScalarPotentials/ElectricPotential.jl

```diff
@@ -40,4 +40,22 @@ function ElectricPotential(nt::NamedTuple)
     N = get_number_of_dimensions(grid)
     ElectricPotential{T, N, S, typeof(grid.axes)}( ustrip.(uconvert.(u"V", nt.values)), grid)
 end
-Base.convert(T::Type{ElectricPotential}, x::NamedTuple) = T(x)
\ No newline at end of file
+Base.convert(T::Type{ElectricPotential}, x::NamedTuple) = T(x)
+
+"""
+    get_ticks_at_positions_of_large_gradient(epot::ElectricPotential)
+
+The electric potential is analyzed in order to find and return ticks where the gradient (electric field) is strong.
+The electric field strength is the strongest at the position of the pn-junction at least for the 1D linear case.

```

Comment:
"The electric field is the strongest..." or "The electric field strength is the largest..."
The electric field strength is also large close to sharp edges of contacts.